enginetest/queries/query_plans.go

```diff
@@ -11400,17 +11610,19 @@ var IntegrationPlanTests = []QueryPlanTest{
 			" │       ├─ cacheable: true\n" +
 			" │       └─ Project\n" +
 			" │           ├─ columns: [E2I7U.id:17!null]\n" +
-			" │           └─ Filter\n" +
-			" │               ├─ (NOT(InSubquery\n" +
-			" │               │   ├─ left: E2I7U.id:17!null\n" +
-			" │               │   └─ right: Subquery\n" +
-			" │               │       ├─ cacheable: true\n" +
-			" │               │       └─ Table\n" +
-			" │               │           ├─ name: AMYXQ\n" +
-			" │               │           └─ columns: [luevy]\n" +
-			" │               │  ))\n" +
-			" │               └─ Table\n" +
-			" │                   └─ name: E2I7U\n" +
+			" │           └─ AntiJoin\n" +
```

Comment:
todo: maybe we need antiHashJoin? which is kind of going to be the original plan, but at least we can choose between options